### ISSUE TITLE
Ps aexww parser does not parser all environment vars correctly

### ISSUE THREAD

**Issue by bfahr** [RedHatInsights/insights-core#1857]
The parser merged in #1848 will parse the examples shown in the parser, but the `get_environ()` method does not correctly parse more complex ps output.  The parser should handle all of the command output in a consistent way, regardless of any possible filtering.

For example a ps line like this:
```
 2365 ?        Ssl    8:52 /usr/bin/docker-containerd-current -l unix:///var/run/docker/libcontainerd/docker-containerd.sock --shim docker-containerd-shim --metrics-interval=0 --start-timeout 2m --state-dir /var/run/docker/libcontainerd/containerd --runtime docker-runc --runtime-args --systemd-cgroup=true REGISTRIES= --add-registry registry.access.redhat.com OPTIONS=--selinux-enabled --log-driver=journald --signature-verification=false DOCKER_HTTP_HOST_COMPAT=1 PATH=/usr/libexec/docker:/usr/bin:/usr/sbin PWD=/ GOTRACEBACK=crash LANG=en_US.UTF-8 DOCKER_NETWORK_OPTIONS= SHLVL=0 DOCKER_CERT_PATH=/etc/docker DOCKER_STORAGE_OPTIONS=
```
is returned by `get_environ` like this:
```
[{'-l': True, 'unix:///var/run/docker/libcontainerd/docker-containerd.sock': True, '--shim': True, 'docker-containerd-shim': True, '--metrics-interval': '0', '--start-timeout': True, '2m': True, '--state-dir': True, '/var/run/docker/libcontainerd/containerd': True, '--runtime': True, 'docker-runc': True, '--runtime-args': True, '--systemd-cgroup': 'true', 'REGISTRIES': '', '--add-registry': True, 'registry.access.redhat.com': True, 'OPTIONS': '--selinux-enabled', '--log-driver': 'journald', '--signature-verification': 'false', 'DOCKER_HTTP_HOST_COMPAT': '1', 'PATH': '/usr/libexec/docker:/usr/bin:/usr/sbin', 'PWD': '/', 'GOTRACEBACK': 'crash', 'LANG': 'en_US.UTF-8', 'DOCKER_NETWORK_OPTIONS': '', 'SHLVL': '0', 'DOCKER_CERT_PATH': '/etc/docker', 'DOCKER_STORAGE_OPTIONS': ''}]
```
Also an environment variable may contain a shell script which includes embedded spaces, and it be parsed by get_environ as a single env variable.

**Comment by shzhou12** [comment]
@bfahr PR to solve this issue is submitted: https://github.com/RedHatInsights/insights-core/pull/1858

**Comment by shzhou12** [comment]
Close this issue for it has been resolved in #1866
